src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
This is a bit weird. Toolbar height changes when width of page reaches 642px and not according to $mdMedia('xs') on 599px.